packages/forge/test/Contest.t.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "@forge-std/Test.sol";
+import "../src/Contest.sol";
+
+contract ContestTest is Test {
+    Contest public contest;
+    uint64 public constant CONTEST_START = 1681650000;
+    uint64 public constant VOTING_DELAY = 10000;
+    uint64 public constant VOTING_PERIOD = 10000;
+    uint64 public constant NUM_ALLOWED_PROPOSAL_SUBMISSIONS = 2;
+    uint64 public constant MAX_PROPOSAL_COUNT = 100;
+    uint64 public constant DOWNVOTING_ALLOWED = 0;
+    uint256 public constant ONE_ETH_IN_WEI = 1000000000000000000;
```

Comment:
I beleive this is `1 ether`: https://docs.soliditylang.org/en/v0.8.19/units-and-global-variables.html#ether-units

Reply:
Yup, I agree! That's what I meant it to be - just denominated in number of wei, is that unclear from the name/should I consider renaming?

Reply:
I meant since there is a built-in `ether` unit in solidity, probably it would make the code even more clear to use that. That way everyone can be sure that it's the right amount of zeros without actually counting them

Reply:
Gotcha! That makes sense, thank you!